apps/test/unit/lib/kits/maker/MakerBoardTest.js

```diff
@@ -14,7 +16,6 @@ import {N_COLOR_LEDS} from '@cdo/apps/lib/kits/maker/PlaygroundConstants';
 /**
  * Run the set of interface conformance tests on the provided class.
  * @param {function} BoardClass
- * @param {function} boardType
  * @param {function} boardSpecificSetup optional
  */
```

Comment:
Weird... why is the doc comment here so far away from the function definition? (To fix later)